packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -95,6 +95,17 @@ const CreateAdForm = () => {
                 }}
             >
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
+                    if (isNaN(values.offer_amount)) {
+                        errors.offer_amount = 'Enter a valid amount';
+                    }
+                    const invalidAmountError = Object.values(errors).includes('Enter a valid amount');
+
+                    if (invalidAmountError) {
+                        Object.entries(errors).forEach(([key, value]) => {
+                            errors[key] = value === 'Enter a valid amount' ? value : undefined;
```

Comment:
I'm struggling a bit to understand this line, if the `value` is not the error we want, we are unsetting it. Why is this needed?

Reply:
It's needed for showing only 'Enter a valid amount' error, when we have that error in the form, because when we have additional errors with that error too, changing values in other inputs will not work, cause of invalid amount in one of the inputs. So that line makes a priority for error showing. When we don't have 'Enter a valid amount' error, we do nothing, cause no additional logic needed.